Python/Setup/PythonTools/PythonToolsFiles.proj

```diff
@@ -135,7 +136,11 @@
     <File Include="$(BuildOutputRoot)Binaries\Analyzer\CompletionDB\*.idb">
       <InstallDirectory>CompletionDB</InstallDirectory>
     </File>
-  
+
+    <File Include="*.idb">
+      <InstallDirectory>CompletionDB</InstallDirectory>
+    </File>
+    

```

Comment:
Oh I see, you've got new default .idb files. Probably best to just let them be analyser normally, but if they're really necessary, put them in the `Analyzer/CompletionDB` folder.